src/index.html

```diff
@@ -1,14 +1,23 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <ul class="container">
+      <li class="block block1">1</li>
+      <li class="block block2">2</li>
+      <li class="block block3">3</li>
+      <li class="block block4">4</li>
+      <li class="block block5">5</li>
+      <li class="block block6">6</li>
```

Comment:
modifiers should be added this way. Not critical as you'll learn it a bit later, but still
```suggestion
      <li class="block block--1">1</li>
      <li class="block block--2">2</li>
      <li class="block block--3">3</li>
      <li class="block block--4">4</li>
      <li class="block block5">5</li>
      <li class="block block6">6</li>
```